Functions.Templates/Bindings/bindings.json

```diff
@@ -1755,6 +1755,17 @@
             "enabledInTryMode": false,
             "documentation": "$content=Documentation\\Token.md",
             "filters": ["MSGraph"],
+            "AADPermissions": [
```

Comment:
I am not sure if the permissions need to be configurable, but if they do need to be we would also need representative text for the scope being defined.